worker/exec/konnector.go

```diff
@@ -383,6 +392,12 @@ func (w *konnectorWorker) ensureFolderToSave(ctx *job.TaskContext, inst *instanc
 			Type: consts.Konnectors,
```

Comment:
Since we can now have 2 references to add to the folder, we should check for their presence on the folder instead of simply checking if none exist.
This way we can recreate a single missing reference.